### ISSUE TITLE
Provenance is inconsistent in how hidden is handled

### ISSUE THREAD

**Issue by jdanish** [netcreateorg/netcreate-itest#131]
For most attributes, hidden impacts 

1. Node attribute display
2. Table display
3. Export / Import

For provenance attributes, they appear to only influence:

3) Export / Import

This is fine for now, but when the other systems are updated and we move towards the new template editor we should revisit this and decide how to make it feel more consistent / change it to be clear even when it is not.

**Comment by benloh** [comment]
@jdanish d56dc8245a32efc5f2d9d4f4771ce5d657addff7 should have fixed this and made this more consistent I think?  (Check out the `dev-dhi` branch.

**Comment by jdanish** [comment]
Yes, that is my understanding / experience thus far. Closing, and will re-open if I see any issues.
